files/blocks/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "blocks",
+  "repo": "astoilkov/jsblocks",
+  "files": {
+    "basePath": "dist/"
+  }
```

Comment:
So people can take a look at the source code. Howerver, it is not that important. I could leave them out if you suggest to do so.